go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/onsi/gomega v1.34.0
 	github.com/operator-framework/api v0.26.0
 	github.com/operator-framework/catalogd v0.20.0
-	github.com/operator-framework/helm-operator-plugins v0.3.1
+	github.com/operator-framework/helm-operator-plugins v0.3.1-0.20240722173643-2e18c5b70f9c
```

Comment:
release?

Reply:
Yeah, talking about it over on Kubernetes Slack, here: https://kubernetes.slack.com/archives/C0181L6JYQ2/p1722268675418209?thread_ts=1722268591.413059&cid=C0181L6JYQ2

Reply:
You're now unblocked here, see below
https://github.com/operator-framework/helm-operator-plugins/releases/tag/v0.4.0